README.md

```diff
@@ -35,40 +137,89 @@ The raw output looks something like this:
     }
 ```
 
-# Installation
+### Further reading
 
-#### Install from source
+- For more info about analytics codes and what the GA-4 rollout means for OSINT: [https://digitalinvestigations.substack.com/p/what-the-rollout-of-google-analytics]
 
-1. Clone repo:
+- For an example investigation usings analytics codes: [https://www.bellingcat.com/resources/how-tos/2015/07/23/unveiling-hidden-connections-with-google-analytics-ids/]
```

Comment:
```suggestion
- For an example investigation usings analytics codes: https://www.bellingcat.com/resources/how-tos/2015/07/23/unveiling-hidden-connections-with-google-analytics-ids/
```
not sure if the brackets were intentional or incomplete markdown, disregard if intentional.